capstone-frontend/src/pages/BorderPage.jsx

```diff
@@ -1,101 +1,105 @@
 import Header from "../components/header";
 import { useParams } from "react-router-dom";
-import { useState, useEffect, act } from "react";
+import { useState, useEffect } from "react";
 import "./BorderPage.css";
+import useSWR from "swr";
+const fetcher = (url) => fetch(url).then((res) => res.json());
+import ModalPopulate from "../components/modal";
 
 function BorderPage() {
   const borderObject = useParams();
   const borderIndex = borderObject.borderid;
-  let specificBorderPosts = [];
 
-  const [crossingData, setCrossingData] = useState([]);
   const [postData, setPostData] = useState({
-    userId: "",
-    borderNum: parseInt(borderIndex),
+    userId: "erramoseduardo@gmail.com",
+    borderNum: "0",
     userInput: "",
```

Comment:
Is this intentionally hard-coded here?

Reply:
Yes, this has just been for my testing: this will be set to the current user's email.